docs/sce_file_contents.md

```diff
@@ -127,6 +130,36 @@ expt_metadata <- metadata(sce)
 | `cellassign_predictions` | If cell typing with `CellAssign` was performed, the full matrix of predictions across cells and cell types. Only present for `processed` objects |
 | `cellassign_reference` | If cell typing with `CellAssign` was performed, is the organ/tissue type for which marker genes were obtained from `PanglaoDB`. Only present for `processed` objects |
 
+### Sample metadata
+
+Relevant sample metadata is available as a data frame stored in the `metadata(sce)$sample_metadata` slot of the `SingleCellExperiment` object.
+Each row in the data frame will correspond to a sample present in the library.
+The following columns are included in the sample metadata data frame for all libraries.
+
+| Column name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `sample_id`   | Sample ID in the form `SCPCS000000`                            |
+| `library_id`   | Library ID in the form `SCPCL000000`                             |
+| `donor_id`  | Unique id corresponding to the donor from which the sample was obtained (equivalent to `particpant_id` included in `single_cell_metadata.tsv`) |
+| `submitter_id`    | Original sample identifier from submitter                      |
+| `submitter`       | Submitter name/id                                              |
+| `age`             | Age at time sample was obtained                                |
+| `sex`             | Sex of patient that the sample was obtained from               |
+| `diagnosis`       | Tumor type                                                     |
+| `subdiagnosis`    | Subcategory of diagnosis or mutation status (if applicable)    |
+| `tissue_location` | Where in the body the tumor sample was located                 |
+| `disease_timing`  | At what stage of disease the sample was obtained, either diagnosis or recurrence |
+| `organism`         | The organism the sample was obtained from (e.g., `Homo_sapiens`) |
+| `development_stage_ontology_term_id` | `HsapDv` ontology term indicating developmental stage - for _Homo sapiens_ either one of the terms for [1 - 11 months old](https://ontobee.org/search?ontology=HSAPDV&keywords=month-old&submit=Search+terms) or for [older than 12 months old](https://ontobee.org/search?ontology=HSAPDV&keywords=year-old&submit=Search+terms). If unavailable, `unknown` is used.  |
```

Comment:
Do we need this level of detail? I would maybe just link to http://obofoundry.org/ontology/hsapdv.html or https://www.ebi.ac.uk/ols/ontologies/hsapdv. The former is probably better for consistency, even if I like EBI's interface a bit better

For our internal use when creating these, we might need the instruction about which terms to use, but when people are looking up the terms, they should not need quite the same level of detail?

```suggestion
| `development_stage_ontology_term_id` | [`HsapDv` ontology](http://obofoundry.org/ontology/hsapdv.html) term indicating developmental stage. If unavailable, `unknown` is used.  |
```